ansible/playbooks/AdoptOpenJDK_Unix_Playbook/roles/Common/tasks/RedHat.yml

```diff
@@ -84,7 +84,7 @@
 - name: Install Java (Not RedHat 6 on ppc64)
   package: "name={{ item }} state=latest"
   with_items: "{{ Java_NOT_RHEL6_PPC64 }}"
-  when: (ansible_distribution_major_version != "6" and ansible_architecture != "ppc64")
```

Comment:
I'm not sure if that's going to produce the desired effect - as it'll knock out the installation on any RHEL7. Did you mean to switch the "and" to an "or" instead?

Reply:
I haven't tested this, but do you know if this will work.
`when: not (ansible_distribution_major_version == "6" and ansible_architecture == "ppc64")`

Reply:
Okay tested now, this syntax does work.

Reply:
Logically that's identical to my suggestion, so I'm ok with that change :-)